app/lib/server/functions/processWebhookMessage.js

```diff
@@ -84,7 +78,16 @@ export const processWebhookMessage = function(messageObj, user, defaultValues =
 			}
 		}
 
-		validateRoomMessagePermissions(room, { uid: user._id, ...user });
+		try {
+			validateRoomMessagePermissions(room, { uid: user._id, ...user });
+		} catch (error) {
+			if (!integration) {
+				throw error;
+			}
+			console.warn(`Warning: The integration "${ integration.name }" failed to send a message to "${ [].concat(integration.channel).join(',') }" because user "${ integration.username }" doesn't have permission or is not part of the channel.`);
```

Comment:
I think a cool down is required otherwise open's logs will be spammed and cause it to be very unreadable.

Reply:
hmm.. ic.. what do you suggest? one log per integration per minute or hour?

Reply:
When is 4.0.0 planned to be released? If next month, then per minute. If not, then per hour or more and maybe that is increased as the versions come along.

Reply:
I liked the idea.. there is no official date for 4.0 yet, I'd expect to be in August. so I went ahead and changed it to show only once per hour. 👍